setup.py

```diff
@@ -33,9 +33,9 @@
     author='Uber AI Labs',
     install_requires=[
         # TODO: pin to a specific version for the release (until JAX's API becomes stable)
-        'jax>=0.2.8',
+        'jax==0.2.10',
         # check min version here: https://github.com/google/jax/blob/master/jax/lib/__init__.py#L26
-        'jaxlib>=0.1.59',
+        'jaxlib==0.1.62',
```

Comment:
why ==

Reply:
maybe time to change `author='Uber AI Labs'`?

Reply:
We can discuss this in the meeting and change it across pyro-ppl repo.

Re `==`: we pinned jax/jaxlib version for each release until jax is stable enough (I thought it was stable but recent regressions changed my mind - we might wait a bit more to relax this restriction)